documentation/qos_monitor/sdd/qos_monitor_sdd.md

```diff
@@ -34,9 +34,81 @@ The QoS Monitor provides the following services:
 The QoS Monitor consumes the following service:
 * QueryAll private service from the ServiceRegistry core system
 
+In case the QoS Monitor started with the DefaultExternal or the OrchestretedExternal stategy, 
+it consumes the "qos-icmp-ping" service.
+
+The "qos-icmp-ping" service is a composit service consisting of a request-response interaction followed by a sequence of events.
+The "qos-icmp-ping" service provider should provide an endpoint of the "qos-icmp-ping" service.
+The input of the "qos-icmp-ping" service endpoint is an IcmpRequest object.
+
+Input IcmpRequest:
+```json
+{
+    "type": "object",
+    "properties": {
+        "timeToRepeat": {
+            "type": "number"
+        },
+        "timeout": {
+            "type": "number"
+        },
+        "packetSize": {
+            "type": "number"
+        },
+        "ttl": {
+            "type": "number"
+        },
+        "host": {
+            "type": "string"
+        }
+    },
+    "required": [
+                "timeToRepeat",
+                "timeout",
+                "packetSize",
+                "ttl",
+                "host"
+            ]
+}
+```
+
+The response of the "qos-icmp-ping" service endpoint is an IcmpResponseACK object.
+
+Output IcmpResponseACK:
+```json
+{
+    "type": "object",
+    "properties": {
+        "externalMeasurementUuid": {
+            "type": "string",
+            "format": "uuid"
+        },
+        "ackOk": {
+            "type": "string"
+        }
+    },
+    "required": [
+                "externalMeasurementUuid",
+                "ackOk"
+            ]
+}
+```
+
+The response must be followed by the publishing of the following events in the order they listed below:
```

Comment:
"by the publishing", maybe the "the" is unnecessary.

Reply:
changed